addons/web_editor/static/src/js/backend.js

```diff
@@ -388,17 +391,12 @@ var FieldTextHtml = AbstractField.extend({
     has_no_value: function () {
         return this.value === false || !this.$content.html() || !this.$content.html().match(/\S/);
     },
-    // FIXME: not sure what this is supposed to do that is still relevant now
-    // commit_value: function () {
-    //     if (this.lang !== 'en_US' && this.$body.find('.o_dirty').length) {
-    //         this.internal_set_value( this.view.datarecord[this.name] );
-    //          this._dirty_flag = false;
-    //          return this.editor.save();
-    //      } else if (this._dirty_flag && this.editor && this.editor.buildingBlock) {
-    //          this.editor.buildingBlock.clean_for_save();
-    //          this.internal_set_value( this.$content.html() );
-    //      }
-    // },
+    commitChanges: function() {
```

Comment:
yes, we need to update the translation terms when user changes content in other language (!= en_us)